presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -395,6 +395,14 @@ Accessors
 
     Returns the great-circle distance between two points on Earth's surface in kilometers.
 
+.. function:: encode_polyline(Geometry) -> varchar
```

Comment:
I'd name these functions `to_encoded_polyline` and `from_encoded_polyline`. A polyline is a general concept, and the current name is ambiguous as to how it's being encoded. In fact, what it's doing is encoding a geometry using Google's "encoded polyline" format.

Reply:
We may want to move these into a separate section, similar to Bing Tiles (@electrum, thoughts)?

Reply:
I moved it to a separate section, let me know if I should revert the change.